### PR TITLE
Fix timer.Remove

### PR DESCRIPTION
**Pull request by GuurgleOS**

### CODE REVIEW

**Comment by zerfgog** [comment]
isn't delay nil here

File: plugins/lua/sand_modules/timer.lua [hunk 1]
```diff
@@ -78,9 +78,9 @@ end
 
 local function Remove( id )
 
-	timers[ id ] = nil
+	timers[ "t"..id ] = nil
 	
-	writepacket(CreateTimerPacket(id, delay, -1));
+	writepacket(CreateTimerPacket("t"..id, delay, -1));
```
Comment (zerfgog) [inline plugins/lua/sand_modules/timer.lua]: isn't delay nil here

**Comment by GuurgleOS** [comment]
I think so, but shouldn't it throw an error then? It works fine by manually adding the "t".

**Comment by zerfgog** [comment]
and in RemoveAll both id and delay are nil, wtf

**Comment by SwadicalRag** [comment]
err wasn't `timer` a part of lau not hash

**Comment by meepdarknessmeep** [comment]
pls pull

**Comment by Jvs34** [comment]
pls pull on my dick

**Comment by SwadicalRag** [comment]
wow

**Comment by SwadicalRag** [comment]
who edited my comment? :angry:

**Comment by mcd1992** [comment]
What's even broken with timer.Remove?

**Comment by zerfgog** [comment]
@mcd1992 did you even look at the change

**Comment by mcd1992** [comment]
R.I.P. I tested it locally and thought it was working.

**Pull request merged**